resources/definitions/fdmprinter.def.json

```diff
@@ -1665,16 +1665,15 @@
                 {
                     "label": "Small Top/Bottom Width",
                     "description": "Small top/bottom regions are filled with walls instead of the default top/bottom pattern. This helps to avoids jerky motions.",
-                    "value": "0",
-                    "default_value": 0,
+                    "value": "skin_line_width * 2",
+                    "default_value": 1,
                     "minimum_value": "0",
-                    "maximum_value_warning": "skin_line_width * 10",
+                    "maximum_value_warning": "skin_line_width * 3",
```

Comment:
I believe this can be increased a bit. I do not see any significant changes till "small top/bottom width"= 3* skin_line_width